packages/same-structure/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@agoric/same-structure",
   "version": "0.0.12-dev.0",
   "description": "Deep equals",
-  "main": "src/sameStructure.js",
+  "main": "index.js",
```

Comment:
Please add `"*.js"` to the `"files"` list.

Reply:
Aha. I see this one. I'll guess that I should just add one like this to marshal/pacjage.json
Done.